### ISSUE TITLE
Not receiving Terminated messages

### ISSUE THREAD

**Issue by fdbeirao** [akkadotnet/akka.net#3567]
I am running akka.net version **1.3.8** installed from nuget, in [**Linqpad 5.31.00**](https://www.linqpad.net/)

I have two actors. A parent and a child. The child is supposed to crash due to its dangerous work. After the parent spawns the child, I use `Context.Watch()` to receive a `Terminated` message when the child actor stops/dies.

When the child actor dies due to a crash/exception, the parent isn't receiving the `Terminated` message. 

What I have tested:

* Both `ReceiveActor` and `UntypedActor`; 
* Doing the spawn+watch either in the constructor or in the `PreStart()`;
* Using the default supervision strategy and the StoppingStrategy;

What I noticed is that I receive the `Terminated` message for graceful stops (`Context.Stop(Self);`) of the child actor, but not for crashes (Exceptions).

Here is a [gist](https://gist.github.com/fdbeirao/d8eb10f85046be853d79a3e447990543) with the smaller example I can come up with.

Please let me know what it is that I am doing wrong.

Kudos for all the work you guys are doing 👍

**Comment by Horusiath** [comment]
@fdbeirao when actor crashes due to exception, it's restarted by default but it doesn't terminate. Termination happens when actor is permanently removed. If you want, you may configure parent's SupervisionStrategy to react with `Directive.Stop` on children exceptions, on which you want to terminate them instead of restarting.

**Comment by fdbeirao** [comment]
@Horusiath thank you for your answer. I guess I must be misinterpreting the `Props.WithSupervisorStrategy(Akka.Actor.SupervisorStrategy.StoppingStrategy)`.. 

Okay I see what I did wrong.. I applied the `.WithSupervisorStrategy(Akka.Actor.SupervisorStrategy.StoppingStrategy)` to the props of the **child**. I should have applied it to the props of the **parent** instead. I just tested it, and indeed now it has the expected behavior.

Thank you so much. Kudos!
